actionsystemtest/lib/action_system_test/server.rb

```diff
@@ -0,0 +1,34 @@
+require "rack/handler/puma"
```

Comment:
Should the puma gem be a runtime dependency if we hard require its Rack handler?